test/controllers/web-payments/handle-auth-response.controller.test.js

```diff
@@ -251,7 +251,7 @@ describe('The web payments handle auth response controller', () => {
     done()
   })
 
-  it('should render the auth_failure view and delete connector response if connector response is in the session and status code is 400', done => {
+  it('should render the auth_failure view and delete connector response if connector response is in the session and status code is 400 Authorisation Rejected', done => {
```

Comment:
Can we add one more test case for a 400 where the error identifier is no AUTHORISATION_REJECTED?